elliottlib/cli/find_builds_cli.py

```diff
@@ -162,7 +161,19 @@ def _fetch_builds_by_id(builds, product_version, session):
 def _fetch_builds_from_diff(from_payload, to_payload, product_version, session):
     green_print('Fetching changed images between payloads...')
     changed_builds = elliottlib.openshiftclient.get_build_list(from_payload, to_payload)
-    return [elliottlib.brew.get_brew_build(b, product_version, session=session) for b in changed_builds]
+    brew_session = koji.ClientSession(constants.BREW_HUB)
+    builds = []
+    for cb in changed_builds:
+        for tag in brew_session.listTags(cb):
```

Comment:
could you add some comments as to _why_ this code is here? frankly i'm having a little trouble following the motivation of this bit, even though i know what the overall goal is.

Reply:
This seems like to get all tags for a Brew build then use the first one in the list. That doesn't seem to correct because if a build is tagged with multiple tags we will don't know which one is desired.